### PR TITLE
store/tikv: do not log slow task if most of the time is wait.

### PR DESCRIPTION
**Pull request by coocood**
We don't care about the slow cop task which most of time is waiting.

### CODE REVIEW

**Comment by coocood** [comment]
@zz-jason @shenli PTAL

**Comment by zz-jason** [comment]
Why you don't care the waiting time?

**Comment by zz-jason** [comment]
I think it's very useful.

**Comment by shenli** [comment]
@zz-jason We need to find the SQL which is really slow and fix it. Other logs will be disturbing.

File: store/tikv/coprocessor.go [hunk 1]
```diff
@@ -627,37 +627,38 @@ func (worker *copIteratorWorker) handleTaskOnce(bo *Backoffer, task *copTask, ch
...
 	if resp.Cop != nil {
 		detail = resp.Cop.ExecDetails
 	} else if resp.CopStream != nil {
 		detail = resp.CopStream.ExecDetails
 	}
+	if detail != nil && detail.HandleTime != nil {
+		processMS = int(detail.HandleTime.ProcessMs)
+		waitMS = int(detail.HandleTime.WaitMs)
+	}
+	if waitMS > (processMS+bo.totalSleep)*minLogWaitFactor {
```
Comment (winkyao) [inline store/tikv/coprocessor.go]: Add some comment here? what minLogWaitFactor means?

**Pull request closed without merging**